src/python/espressomd/lb.pyx

```diff
@@ -542,3 +548,144 @@ cdef class LBFluidRoutines:
 
         def __set__(self, value):
             raise NotImplementedError
+
+cdef class LBSlice:
```

Comment:
is there a drawback if it is? its more convenient to handle types of cython and python arrays if both classes are cython classes.

Reply:
In general we try to avoid cython classes if possible. Cython is only used to get bindings to the c++ core interface. The  interface to this C++ interface is implemented in the ```LBFluid``` class. So to have a better separation of concerns you should not use a cython class here and it should also not be necessary.

Reply:
sorry if I'm uneducated about that (I'm not that familiar with mixing python and cython), but is it even possible to simply call a cython class from a python one, like i do it here? as far as i know there is no proper workaround for that

Reply:
> sorry if I'm uneducated about that (I'm not that familiar with mixing python and cython), but is it even possible to simply call a cython class from a python one, like i do it here? as far as i know there is no proper workaround for that

in line wit @jngrad comment

Reply:
we are only accessing LB fluid properties that are already interfaced or do we want to change the way we retrieve information from the c++ interface?

Reply:
> i think this does not need to be a cython class

I'm currently working on a Python version of the Cython class, I'll push the commits once I'm done updating the test.

Reply:
This is now a regular Python class.